src/app/components/work-item-create/work-item-create-selector/work-item-create-selector.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, Input, OnChanges, EventEmitter, Output } from '@angular/core';
+import { Component, OnInit, Input, AfterViewChecked, OnChanges, EventEmitter, Output, ViewChild, ElementRef, Renderer2 } from '@angular/core';
```

Comment:
It exceeds the standard number of columns. Can you break it into lines?

Reply:
sure